apps/i18n/netsim/en_us.json

```diff
@@ -18,6 +18,16 @@
   "hexadecimal": "Hexadecimal",
   "infinity": "Infinity",
   "instructions": "Instructions",
+  "instructions_AddressingMessages": "In this version of the simulator, you will connect to a Router instead of directly to another student. Enter an address in the \"To\" box to send a message to another student.",
+  "instructions_AutomaticDNS": "In this version of the simulator a DNS service is provided for you. Send a message in the form GET [hostname] to the DNS service to get a student's address.",
+  "instructions_default": "You have loaded NetSim in superuser mode.",
+  "instructions_SendAB": "UNDER CONSTRUCTION: This version of NetSim is not yet functional. It will allow students to manually set/read individual As and Bs on a simplex wire.",
+  "instructions_SendAscii": "UNDER CONSTRUCTION: This version of NetSim is not yet functional. It will allow students to send ASCII to each other.",
+  "instructions_SendBits": "UNDER CONSTRUCTION: This version of NetSim is not yet functional. It will allow students to send binary as zeroes and ones to one another",
```

Comment:
Nit: It might be worth putting public-ready instructions here even if you plan on changing them later.  That way if they accidentally go live it won't say "Under Construction".